libs/wingc/src/file_graph.rs

```diff
@@ -44,6 +44,17 @@ impl FileGraph {
 		self.path_to_node_index.contains_key(path)
 	}
 
+	/// Returns a list of the direct dependencies of the given file
+	/// (does not include all transitive dependencies)
```

Comment:
maybe add a comment indicating that `path` must be relative to to the root of the file graph. otherwise this will panic.